component-runtime-impl/src/main/java/org/talend/sdk/component/runtime/record/SchemaImpl.java

```diff
@@ -47,12 +62,14 @@
 
         private List<Entry> entries = new ArrayList<>();
 
+        private Map<String, String> props = new LinkedHashMap<>(0);
```

Comment:
While use linkedHashMap here ? really need to keep props order ?

Reply:
Yes, this is deliberated! This choice is relevant to Avro schema wrapping, internally `Schema` uses the `LinkedHashMap` class for its props.